### ISSUE TITLE
Add secret options to usage hint

### ISSUE THREAD

**Issue by mk-pmb** [reid/node-jslint#159]
From browsing the issues, I leaned that I'll want `--fudge` and `--es6` from now on, so I'd like my wrapper script to be able to detect their availability. Could you add them to the usage hint?

```bash
$ jslint --help 2>&1 | grep -oPe '\[--(f|es)\S+'
[--es5]
[--filter]
[--forin]
$ jslint --version
node-jslint version: 0.10.1  JSLint edition 2013-08-26
```

**Comment by smikes** [comment]
I think you need

jslint --edition=es6 --fudge

**Comment by mk-pmb** [comment]
Yes, but only in versions that support `--fudge`. Those versions shall advertize their support for `--fudge` in their `--help` options list, so my wrappper can detect whether it's supported.

**Comment by smikes** [comment]
Are you sure you want to consume the interface that we provide here?  If you're programmatically using Doug Crockford's jslint interface -- https://github.com/douglascrockford/jslint -- then you don't have to work around me :-)

**Comment by mk-pmb** [comment]
In my case I like the CLI more, because the wrapper script is written in bash and aggregates output from other linters (e.h. PHP syntax check) as well.
